avm/res/azure-stack-hci/cluster/main.bicep

```
@@ -185,6 +185,9 @@ output systemAssignedMIPrincipalId string = cluster.identity.principalId
 @description('The location of the cluster.')
 output location string = cluster.location
 
+@description('The name of the custom location.')
+output customLocationName string = deploymentSettings!.customLocationName
```

Please note that this will very likely fail if one deploys the cluster without deployment settings (which are, afterall, optional)
It might be better to use a syntax like
```suggestion
output customLocationName string? = (deploymentSettings ?? {}).?customLocationName
```

---

Hey @xhy8759, is this still being worked on?

---

@xhy8759 in case you forgot about this one ;-)